modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -70,11 +70,26 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^]:
+. Add `* file:` to `default-src` in the `Content-Security-Policy` meta tag inside the `index.html` file:
++
+[source,html]
+----
+<meta http-equiv="Content-Security-Policy" content="default-src * file: 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *; img-src 'self' data: content:;">
+----
+. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^]:
++
 [source,bash]
 ----
 $ npm install @aerogear/auth
 ----
+. Install the `cordova-plugin-inappbrowser` plugin
++
+[source,bash]
+----
+$ cordova plugin add cordova-plugin-inappbrowser
+----
+. Use e.g `browserify` so that you can use the `require` statement
+. In the `init` snippet use `require` instead of `import`
```

Comment:
Maybe let's remove the above 2 lines, user will probably know how to import libraries and borwserify is probably not preferred way.

Reply:
👍